### DISCUSSION TITLE
An interactive portfolio website inspired by the macOS interface

### DISCUSSION THREAD

**Discussion by ansxuman** [janosh/awesome-sveltekit#131] [Show and tell]
Hey, I’ve developed an interactive portfolio website inspired by the macOS interface, built using SvelteKit, TailwindCSS, and TypeScript. The project is hosted on Netlify and a Raspberry Pi 5. I was wondering if this project would be a good fit to include in this repository. 

Project : https://github.com/ansxuman/macOS-Themed-Portfolio 
Website: https://ssh-i.in/ || https://macosportfolio.netlify.app

**Comment by janosh** [comment]
sure, happy to accept a PR
